recipe-server/client/control_new/components/recipes/ApprovalForm.js

```diff
@@ -0,0 +1,52 @@
+import { Button, Form, Input } from 'antd';
+import autobind from 'autobind-decorator';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import FormActions from 'control_new/components/common/FormActions';
+import { createForm, FormItem } from 'control_new/components/common/forms';
+
+@createForm({
+  validateFields(values) {
```

Comment:
nope, that's left over from when I was trying to figure out how to capture which button was clicked.